libdeno/binding.cc

```diff
@@ -13,6 +13,12 @@
 
 #define GLOBAL_IMPORT_BUF_SIZE 1024
 
+#ifdef _WIN32
+#include <malloc.h>
```

Comment:
According to https://stackoverflow.com/questions/12973311/difference-between-stdlib-h-and-malloc-h, I'd suggest to use `<stdlib.h>` instead. It is only available when you are compiling in MSVC, but not in clang.

Reply:
thx